test/e2e/basic_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	adminIAMPolicyYaml string = "../resources/iam-policy-default.yaml"
+	adminIAMPolicyName string = "rule-of-two"
+	adminCRBYaml       string = "../resources/admin-crb.yaml"
+	otherIAMPolicyYaml string = "../resources/iam-policy-other.yaml"
+	otherIAMPolicyName string = "there-can-be-only-one"
+	otherCRBYaml       string = "../resources/other-crb.yaml"
+	otherRoleYaml      string = "../resources/other-clusterrole.yaml"
+)
+
+var _ = Describe("Test IAM policy", Ordered, func() {
+	Describe("Test default clusterrole (cluster-admin)", func() {
```

Comment:
Nit: Should these tests be in a loop since at quick glance I think they're doing the same/similar things?

Reply:
It didn't feel quite right in a loop/table, but I was able to pull some of the common stuff out. Let me know what you think of this approach (and thanks for the suggestion :).